flake.nix

```diff
@@ -10,7 +10,7 @@
     clickhouse-haskell.url = "github:piyushKumar-1/clickhouse-haskell";
     clickhouse-haskell.flake = false;
 
-    euler-hs.url = "github:juspay/euler-hs";
+    euler-hs.url = "github:piyushKumar-1/euler-hs/464ab99327e44bd5eee78a9986f2f9abc5f160de";
```

Comment:
Could you open a PR on https://github.com/juspay/euler-hs and link it here?

Also you might want to fork it on nammayatri org.

Reply:
If we can get it merged (@aravindgopall) we don't even need to change `flake.nix` here (just the lock file).

Reply:
yeah, m testing the changes rn. will make those changes once validated it working fine.